src/backend/distributed/shared_library_init.c

```diff
@@ -689,6 +689,18 @@ RegisterCitusConfigVariables(void)
 		GUC_STANDARD,
 		NULL, NULL, NULL);
 
+	DefineCustomBoolVariable(
+		"citus.disable_manual_changes_to_shards",
```

Comment:
Maybe enable_manual_changes_to_shards to be consistent with Postgres GUC naming.